cmd/minikube/cmd/start.go

```diff
@@ -136,15 +137,21 @@ func runStart(cmd *cobra.Command, args []string) {
 	kubeHost = strings.Replace(kubeHost, ":2376", ":"+strconv.Itoa(constants.APIServerPort), -1)
 
 	// setup kubeconfig
+	keepContext := viper.GetBool(keepContext)
 	name := constants.MinikubeContext
 	certAuth := constants.MakeMiniPath("ca.crt")
 	clientCert := constants.MakeMiniPath("apiserver.crt")
 	clientKey := constants.MakeMiniPath("apiserver.key")
-	if err := setupKubeconfig(name, kubeHost, certAuth, clientCert, clientKey); err != nil {
+	if err := setupKubeconfig(name, kubeHost, certAuth, clientCert, clientKey, keepContext); err != nil {
 		glog.Errorln("Error setting up kubeconfig: ", err)
 		cmdUtil.MaybeReportErrorAndExit(err)
 	}
-	fmt.Println("Kubectl is now configured to use the cluster.")
+
+	if keepContext {
+		fmt.Println("The local Kubernetes cluster has started. The kubectl context has not been alterted, kubectl will require --context or --kubeconfig to use the local Kubernetes cluster.")
```

Comment:
I don't think --kubeconfig could be used unless the user did something like create a minikube context in a different kubeconfig. That seems ... unusual. If they did that then I guess they would know what they did and what they needed to do.

I'll remove the kubeconfig suggestion and change the wording on --context.